android/assets/jsons/translations/template.properties

```diff
@@ -847,6 +847,7 @@ Obsolete with [obsoleteTech] =
 Occurs on [listOfTerrains] = 
 Placed on [terrainType] = 
 Can be found on  = 
+Can only be created by Mercantile City-States = 
```

Comment:
No need, TranslationFileWriter will pick this up.

Reply:
Just this needs to be removed and it's good to merge :)